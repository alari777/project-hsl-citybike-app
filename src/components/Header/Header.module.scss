.header {
  background-image: linear-gradient(
                  to bottom,
                  rgba(0, 0, 0, 0.7),
                  rgba(0, 0, 0, 0.5)
  );
  display: flex;
  align-items: center;
  padding: 0 1.25rem;
  min-height: 5rem;
}
