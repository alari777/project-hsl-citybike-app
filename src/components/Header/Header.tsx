import { FC } from 'react';
import styles from '@/components/Header/Header.module.scss';

const Header: FC = () => {
  return <header className={styles.header}>Helsinki city bike</header>;
};

export default Header;
