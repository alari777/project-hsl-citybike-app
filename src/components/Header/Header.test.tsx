import { render, screen } from '@testing-library/react';
import Header from './Header';

describe('should render header', () => {
  beforeEach(() => {
    render(<Header />);
  });

  test('should render trips link in header navbar', () => {
    const navTrips = screen.getByTestId('nav-trips');
    expect(navTrips).toBeInTheDocument();
    expect(navTrips).toMatchSnapshot();
  });
});
