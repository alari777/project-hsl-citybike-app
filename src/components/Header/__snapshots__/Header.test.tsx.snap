// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render header should render manage link in header navbar 1`] = `
<a
  class="nav-item nav-link"
  data-testid="nav-manage"
  href="/manage"
>
  Manage: trips and stations
</a>
`;

exports[`should render header should render stations link in header navbar 1`] = `
<a
  class="nav-item nav-link"
  data-testid="nav-stations"
  href="/stations"
>
  Stations
</a>
`;

exports[`should render header should render trips link in header navbar 1`] = `
<a
  class="nav-item nav-link active"
  data-testid="nav-trips"
  href="/"
>
  Trips
</a>
`;
